docs/installation.rst

```diff
@@ -68,17 +88,9 @@ For mediaarea tools, it can be easiest to use their own snapshot repository:
 
 ``wget https://mediaarea.net/repo/deb/repo-mediaarea-snapshots_1.0-13_all.deb && sudo dpkg -i repo-mediaarea-snapshots_1.0-13_all.deb && sudo apt update && sudo apt install mediainfo dvrescue qcli rawcooked mediaconch``
 
-Using pip/setup.py
-------------------
-
-the following is currently experimental, but it should work fine:
-
-You can get a selection of scripts by making sure that ``pip`` installed, then running:
-``pip install ifiscripts``
-or ``cd`` into the ``ifiscripts`` cloned folder and run
-``pip install .``
-
-You may need to run the ``pip`` commands with ``sudo``.
 
-The pip installation methods have the added benefit of installing the python dependencies such as ``lxml``.
+----------
 
+Please find an extra `installation and update guideline 
+<https://ifi1-my.sharepoint.com/:b:/g/personal/yhe_irishfilm_ie/ES5_i9-khIpBlH8GDVhy5eMB846bDQrHpJajFjJeKuJyhQ?e=bNIm8H>`_
```

Comment:
wow, these are great at a glance, should the content of this link be in this installation file instead of the current contents?

Reply:
Not sure yet. The current contents are also helpful for advanced users. This doc is too basic and I'm not sure if people really need it.